app/src/main/java/org/simple/clinic/monthlyreport/form/QuestionnaireEntryScreen.kt

```diff
@@ -149,14 +152,20 @@ class QuestionnaireEntryScreen : BaseScreen<
   }
 
   private fun setToolbarMonth(response: QuestionnaireResponse) {
+    val stringResource = when (screenKey.questionnaireType) {
+      is MonthlyScreeningReports -> R.string.monthly_screening_reports_screening_report
+      is MonthlySuppliesReports -> R.string.monthly_supplies_report_supplies_report
+      is MonthlyDrugReports -> R.string.monthly_drug_stock_reports_drug_stock_reports
+      else -> R.string.monthly_screening_reports_screening_report
```

Comment:
Keep it empty or `undefined` in the default condition. 
Also, can we raise an error to Sentry?